pom.xml

```diff
@@ -1309,23 +1309,16 @@
     </profile>
 
     <profile>
-      <id>postgresql</id>
+      <id>production</id>
```

Comment:
@pgier @rnc @matejonnet @vibe13 The default db s H2 now. I've also added a `production` profile.